integr8ly/ci-cd
# created browser-based-testsuite-pipeline definition

## Motivation
https://issues.jboss.org/browse/INTLY-558

## What
Create jjb for browser-based-testsuite-pipeline

## Verification Steps
The updated pipeline was built here (the last test failed, but this is not related to the pipeline itself): 

https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/browser-based-testsuite-pipeline/141/

Additional check is to run this command inside the repository with PR changes to confirm that the update is successful (and also make sure that the pipeline configuration does not change -> as the template is the same as definition on Jenkins): 

`jenkins-jobs --conf jenkins_jobs.ini update jobs/integr8ly/browser-based-testsuite-pipeline.yaml`

## Checklist:


## Progress
